packages/web/src/pages/rewards-page/Tiers.tsx

```diff
@@ -282,40 +283,59 @@ const TierColumn = ({
       />
       {(
         Object.keys(messages.features) as Array<keyof typeof messages.features>
-      ).map((feature) => (
-        <Flex key={feature} pv='m' borderTop='default' justifyContent='center'>
-          <Text>
-            {tierFeatures[feature] ? (
-              <Flex h={24} direction='row' alignItems='center' gap='m'>
-                <IconValidationCheck />
-                {feature === 'customDiscordRole' && current ? (
-                  <Tooltip text='Refresh Discord role'>
-                    <Button
-                      size='small'
-                      variant='secondary'
-                      iconLeft={IconRefresh}
-                      onClick={onClickDiscord}
-                    />
-                  </Tooltip>
-                ) : null}
-              </Flex>
-            ) : (
-              <Flex h={24} w={24} alignItems='center' justifyContent='center'>
-                <Flex
-                  h={16}
-                  w={16}
-                  borderRadius='circle'
-                  border='strong'
-                  css={{
-                    borderWidth: 2,
-                    borderColor: 'var(--harmony-neutral)'
-                  }}
-                />
-              </Flex>
-            )}
-          </Text>
-        </Flex>
-      ))}
+      ).map((feature) => {
+        const minAudio =
+          badgeTiers.find((b) => b.tier === tier)?.minAudio.toString() ?? '0'
```

Comment:
think we can keep this as a number. `formatNumberCommas` takes in number values and converts to string. then we can check `!minAudio` below

Reply:
it's a BN, not a number, so unf going to string is needed here. we want to get off of BN for BigInt, but that's a lift of sorts